lib/index.js

```diff
@@ -38,20 +44,30 @@ module.exports = function (gulp, options) {
   }
 
   var lintOptions = options.lint;
-  if (!lintOptions.files) {
-    lintOptions.files = [
-      options.sourceFiles,
-      options.unitTestFiles,
-      options.integrationTestFiles,
-      'index.js',
-      'main.js',
-      'gulpfile.js',
-      'gulpfile.babel.js',
-      'config/**/*.json',
-      'lib/**/*.json',
-      'test/**/*.json'
-    ];
-  }
+  lintOptions.files = lintOptions.files || [
+    options.sourceFiles,
+    options.unitTestFiles,
+    options.integrationTestFiles,
+    'index.js',
+    'main.js',
+    'gulpfile.js',
+    'gulpfile.babel.js',
+    'config/**/*.json',
+    'config/**/*.js',
+    'lib/**/*.json',
+    'test/**/*.json'
```

Comment:
I should of brought this up earlier, but sometimes, fixtures may on purposes have poorly formed json data (IE, to reflect an API that doesn't actually have good json standards like in the style guide), I would exclude testing like all *.json under a fixtures folder for example to remedy this in test.